spec/swagger.yaml

```
@@ -73,41 +70,31 @@ schemes:
 consumes:
   - application/json; charset=utf-8
 produces:
-  - application/json; charset=utf-8
+  - application/json; charset=utf-8  
 securityDefinitions:
-  bearer:
-    type: apiKey
-    name: Authorization
-    in: header
-    description: >
-      Для авторизации вызовов мы используем [JWT](https://jwt.io).
-      Cоответствующий ключ передается в заголовке.
-
-      ```shell
-       Authorization: Bearer {JWT}
-      ```
-
-      Посмотреть ваш API-ключ вы можете в [личном
-      кабинете](https://dashboard.rbk.money/).
-
-
-      Обратите внимание, что API-ключ должен использоваться исключительно для межсерверного взаимодействия.
-
-      Для авторизации запросов, выполняемых из контекста браузера плательщика (например, токенизация платежных данных методом [createPaymentResource](https://developer.rbk.money/api/#operation/createPaymentResource)) используйте полученный при вызове [createInvoice](https://developer.rbk.money/api/#operation/createInvoice) `invoiceAccessToken`, либо создайте его напрямую, вызвав метод [createInvoiceAccessToken](https://developer.rbk.money/api/#operation/createInvoiceAccessToken).
-
-
-      При попытке вызова [createPaymentResource](https://developer.rbk.money/api/#operation/createPaymentResource) с ключом, отличным от invoiceAccessToken, платформа вернет ошибку авторизации со статусом HTTP/401.
-
-
-      Ключи не разделяются на тестовые и боевые, ваш API ключ открывает доступ
-      ко всем функциям платформы. Для тестовых транзакций используйте ID
-      тестовых магазинов.
-
-
-
-
+   bearer:
+     type: apiKey
+     name: Authorization
+     in: header
+     description: >
```

Быть может сделать
```suggestion
     description: |
```
и удалить все эти излишние `<br/>`?

---

ок